lib/core/versions/latest/models/DocumentModel.ts

```diff
@@ -6,6 +6,6 @@ import ServiceEndpointModel from './ServiceEndpointModel';
  * NOTE: This model should ONLY be used by the `DocumentComposer`.
  */
 export default interface DocumentModel {
-  public_keys?: PublicKeyModel[];
-  service_endpoints?: ServiceEndpointModel[];
+  publicKey?: PublicKeyModel[];
+  service?: ServiceEndpointModel[];
```

Comment:
Similar comment to `VerificationRelationship`, this "misalignment" if you will, is intentional. Further discussion needed if you disagree with it, but this is not strictly related to the snake_casing change. So I'd suggest to change them to `publicKeys` and `serviceEndpoints` respectively to get the casing portion merged without further delay.

Reply:
-1 to making this reference implementation internal data model use an `s` and the external data model not. needlessly different, will result in more implementation bugs.

Reply:
I disagree. But since this is contentious and technically unrelated to the casing change, please leave this part of the change out for a separate discussion so we can at least push the agreed upon change as at least a small win.

Reply:
Seems like the consensus on the call was to take these changes.